src/pages/organizers/[organizerId]/preview/RequestOwnershipModal.tsx

```diff
@@ -0,0 +1,117 @@
+import { Trans, useTranslation } from 'react-i18next';
+import { useQueryClient, UseQueryResult } from 'react-query';
+
+import { useRequestOwnershipMutation } from '@/hooks/api/ownerships';
+import { useGetUserQuery, User } from '@/hooks/api/user';
+import { SupportedLanguage } from '@/i18n/index';
+import { Organizer } from '@/types/Organizer';
+import { Alert, AlertVariants } from '@/ui/Alert';
+import { Box } from '@/ui/Box';
+import { Modal, ModalSizes, ModalVariants } from '@/ui/Modal';
+import { FetchError } from '@/utils/fetchFromApi';
+import { getLanguageObjectOrFallback } from '@/utils/getLanguageObjectOrFallback';
+
+type Props = {
+  organizer: Organizer;
+  isVisible: boolean;
+  onClose: () => void;
+  onSuccess?: () => void;
+  onError?: () => void;
+};
+
+const RequestOwnershipModal = ({
+  organizer,
+  isVisible,
+  onClose,
+  onSuccess,
+  onError,
+}: Props) => {
+  const { t, i18n } = useTranslation();
+  const queryClient = useQueryClient();
+  const router = useRouter();
+  const organizerId = router.query.organizerId as string;
+  const [isSuccessAlertVisible, setIsSuccessAlertVisible] = useState(false);
+  const [isErrorAlertVisible, setIsErrorAlertVisible] = useState(false);
+
+  const organizerName: string = getLanguageObjectOrFallback(
+    organizer?.name,
+    i18n.language as SupportedLanguage,
+    organizer?.mainLanguage as SupportedLanguage,
+  );
+
+  const getUserQuery = useGetUserQuery() as UseQueryResult<User, FetchError>;
+
+  const userId = getUserQuery.data?.sub;
+
+  const requestOwnershipMutation = useRequestOwnershipMutation({
+    onSuccess: async () => {
+      await queryClient.invalidateQueries('ownership-requests');
+      onClose();
+      setIsSuccessAlertVisible(true);
+      if (onSuccess) onSuccess();
+    },
+    onError: () => {
+      onClose();
+      setIsErrorAlertVisible(true);
+      if (onError) onError();
+    },
+  });
+
+  return (
+    <>
+      <Modal
+        title={t('organizers.ownerships.request.confirm_modal.title', {
+          organizerName,
+        })}
+        confirmTitle={t('organizers.ownerships.request.confirm_modal.confirm')}
+        cancelTitle={t('organizers.ownerships.request.confirm_modal.cancel')}
+        visible={isVisible}
+        variant={ModalVariants.QUESTION}
+        onClose={onClose}
+        onConfirm={() => {
+          requestOwnershipMutation.mutate({
+            itemId: organizerId,
+            ownerId: userId,
+          });
+        }}
+        size={ModalSizes.MD}
+      >
+        <Box padding={4}>
+          <Trans
+            i18nKey="organizers.ownerships.request.confirm_modal.body"
+            values={{ organizerName }}
+          />
+        </Box>
+      </Modal>
+      {isSuccessAlertVisible && (
+        <Alert
+          variant={AlertVariants.SUCCESS}
+          marginBottom={4}
+          closable
+          fullWidth
+          onClose={() => setIsSuccessAlertVisible(false)}
+        >
+          <Trans
```

Comment:
Is there a reason why we use "**Trans**" here in a lot of places?
Can we not use the regular syntax?
```
{t(organizers.ownerships.request.success, {organizerName)}
```

Reply:
Some do some more advanced stuff so sometimes you need it but I think this is just a leftover from a copy from somewhere else, I prefer the string version too, will switch it